### PR TITLE
Update zh_cn.lang

### PR DESCRIPTION
**Pull request by Snownee**

### CODE REVIEW

**Comment by mezz** [comment]
Thank you!
@3TUSK can you review this?

File: src/main/resources/assets/jei/lang/zh_cn.lang [hunk 1]
```diff
@@ -1,62 +1,54 @@
...
 jei.tooltip.cheat.mode=作弊模式
 jei.tooltip.recipe.by=该配方由 %s 添加
-
 # Error Tooltips
 jei.tooltip.error.recipe.transfer.missing=缺失物品
 jei.tooltip.error.recipe.transfer.inventory.full=物品栏已满
 jei.tooltip.error.recipe.transfer.unknown=未知错误，请查阅日志获得详细信息
 jei.tooltip.error.recipe.transfer.no.server=服务器必须安装 JEI
-jei.tooltip.error.recipe.transfer.too.large.player.inventory=合成表不是2×2的
+jei.tooltip.error.recipe.transfer.too.large.player.inventory=配方不是 2×2 的
```
Comment (liach) [inline src/main/resources/assets/jei/lang/zh_cn.lang:20]: 最好表示此配方在2*2的格子里面装不下

Comment (Snownee) [reply src/main/resources/assets/jei/lang/zh_cn.lang:20]: 然而原文如此，建议你先让原文改过来

**Comment by liach** [comment]
@mezz https://github.com/mezz/JustEnoughItems/blob/1.12/src/main/resources/assets/jei/lang/en_us.lang#L20
I suggest you directly pointing out that the recipe is too large in this entry.

**Comment by mezz** [comment]
The original English localization for that is bad, I agree. I will think of something better.

**Comment by liach** [comment]
Thanks!

File: src/main/resources/assets/jei/lang/zh_cn.lang [hunk 2]
```diff
@@ -107,11 +95,26 @@ gui.jei.category.fuel=燃料
...
+config.jei.search.tooltipSearchMode.comment=当按提示搜索时，要求使用 "#" 作为前缀
+config.jei.search.oreDictSearchMode=$矿物词典
+config.jei.search.oreDictSearchMode.comment=当按矿物词典搜索时，要求使用 "$" 作为前缀
+config.jei.search.creativeTabSearchMode=%%创造标签页
+config.jei.search.creativeTabSearchMode.comment=当按创造标签页名搜索时，要求使用 "%" 作为前缀
+config.jei.search.colorSearchMode=^颜色
+config.jei.search.colorSearchMode.comment=当按物品颜色搜索时，要求使用 "^" 作为前缀
+config.jei.advanced.giveMode=拿取模式
+config.jei.advanced.giveMode.comment=选择 JEI 会直接将物品移入物品栏或是用鼠标将其捡起。
+# DEBUG (for debug mode, do not need translation)
```
Comment (3TUSK) [inline src/main/resources/assets/jei/lang/zh_cn.lang]: 117 - 120 行可以直接删掉了

File: src/main/resources/assets/jei/lang/zh_cn.lang [hunk 3]
```diff
@@ -107,11 +95,26 @@ gui.jei.category.fuel=燃料
...
+gui.jei.category.brewing=酿造
 gui.jei.category.brewing.steps=步数：%s
-gui.jei.category.itemInformation=描述
-
+gui.jei.category.itemInformation=信息
 # DEBUG （调试用条目，不需翻译，请不要将英文条目重新复制回来）
 #
 #
 #
+config.jei.search.modNameSearchMode=@模组名称
```
Comment (3TUSK) [inline src/main/resources/assets/jei/lang/zh_cn.lang]: 105 - 116 行位置不对，什么情况？

**Review by 3TUSK — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/jei/lang/zh_cn.lang (hunk 3 above)
Comment (Snownee) [reply src/main/resources/assets/jei/lang/zh_cn.lang]: 脚本问题，具体原因不得而知

**Comment by Snownee** [comment]
@3TUSK 
手动整理了一下

File: src/main/resources/assets/jei/lang/zh_cn.lang [hunk 4]
```diff
@@ -107,11 +109,6 @@ gui.jei.category.fuel=燃料
 gui.jei.category.fuel.smeltCount.single=烧炼 1 个物品
 gui.jei.category.fuel.smeltCount=烧炼 %s 个物品
 gui.jei.category.fuel.burnTime=燃烧时间：%s
-gui.jei.category.brewing=酿造配方
+gui.jei.category.brewing=酿造
 gui.jei.category.brewing.steps=步数：%s
-gui.jei.category.itemInformation=描述
-
-# DEBUG （调试用条目，不需翻译，请不要将英文条目重新复制回来）
```
Comment (3TUSK) [inline src/main/resources/assets/jei/lang/zh_cn.lang]: 还是建议把这几行按原来的方式留空...

**Review by 3TUSK — APPROVED** [review]

**Review by liach — APPROVED** [review]

**Pull request merged**